scala-xml/src/main/scala/org/http4s/scalaxml/ElemInstances.scala

```diff
@@ -18,19 +18,19 @@ package org.http4s
 package scalaxml
 
 import cats.effect.Concurrent
+import cats.syntax.all._
+import fs2.data.xml.XmlException
+import fs2.data.xml.scalaXml._
 import org.http4s.Charset.`UTF-8`
 import org.http4s.headers.`Content-Type`
 
-import java.io.ByteArrayInputStream
 import java.io.StringWriter
 import javax.xml.parsers.SAXParserFactory
-import scala.util.control.NonFatal
 import scala.xml.Elem
-import scala.xml.InputSource
-import scala.xml.SAXParseException
 import scala.xml.XML
 
 trait ElemInstances {
+  @deprecated("0.23.12", "This factory is no longer used")
   protected def saxFactory: SAXParserFactory
```

Comment:
This is the (semantically) breaking change.

Reply:
Replacing SAX will also change the types of errors that are raised.